src/main/java/net/bramp/ffmpeg/probe/FFmpegStream.java

```diff
@@ -65,4 +65,10 @@ public enum CodecType {
   public FFmpegDisposition disposition;
 
   public Map<String, String> tags;
+  public SideDataList[] side_data_list;
+
+  public static class SideDataList {
+
+    public int rotation;
+  }
```

Comment:
Should we also add side_data_type and displaymatrix?

Reply:
Side_data_type and display matrix were excluded because they were rarely used. But come to think of it, it doesn't seem bad to add. Thanks